### PR TITLE
[Live-Site] - fix to run on local enviroment

### PR DESCRIPTION
**Pull request by Pratiyushkumar**
This fix is due to every time I go to the URL.js file and change the URL for the development environment and when raising the PR I need to discard the change. so that's why raising the PR to fix the issue.

### CODE REVIEW

**Review by shubhamsigdar1 — APPROVED** [review]

**Review by iamitprakash — DISMISSED** [review]

**Review by satyam73 — COMMENTED** [review]
Why the prod one got changes to localhost:3000?

**Comment by Pratiyushkumar** [comment]
> Why the prod one got changes to localhost:3000?

image
it's inside development not prod

**Review by shubhamsigdar1 — DISMISSED** [review]

File: app/constants/urls.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import ENV from 'website-www/config/environment';
 
 const SCHEME = 'https://';
+const DEVELOPMENT_SCHEME = 'http://';
 const DOMAIN = 'realdevsquad.com';
+const DEVELOPMENT = 'localhost:3000';
```
Comment (satyam73) [inline app/constants/urls.js:6]: DEVELOPMENT_DOMAIN will be better

**Review by satyam73 — CHANGES_REQUESTED** [review]

**Review by satyam73 — APPROVED** [review]
LGTM

**Review by shubhamsigdar1 — APPROVED** [review]

**Pull request merged**